app/v2/models/response/retrieveBalanceAndTransactions/FinanceDetails.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveBalanceAndTransactions
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import v2.models.response.retrieveBalanceAndTransactions.financeDetailTypes.mapping
+
+case class FinanceDetails(taxYear: String,
+                          documentId: String,
+                          chargeType: Option[String],
+                          mainType: Option[String],
+                          taxPeriodFrom: Option[String],
+                          taxPeriodTo: Option[String],
+                          contractAccountCategory: Option[String],
+                          contractAccount: Option[String],
+                          documentNumber: Option[String],
+                          documentNumberItem: Option[String],
+                          chargeReference: Option[String],
+                          mainTransaction: Option[String],
+                          subTransaction: Option[String],
+                          originalAmount: Option[BigDecimal],
+                          outstandingAmount: Option[BigDecimal],
+                          clearedAmount: Option[BigDecimal],
+                          accruedInterest: Option[BigDecimal],
+                          items: Seq[FinancialDetailsItem])
+
+object FinanceDetails {
+
+  implicit val reads: Reads[FinanceDetails] = (
+    (JsPath \ "taxYear").read[String] and
```

Comment:
The spec if not very specific about this but we need to convert taxYear to MTD format i.e. YYYY-YY, e.g. as we do in ChargeHistoryDetail.